src/components/common/prev-next/PrevNextSection.js

```diff
@@ -20,13 +20,22 @@ const PrevNextSection = ({ sidebar, location, next }) => {
 
 		const { groups } = sidebarfile;
 		const flatSidebar = [];
-
+		const isRSDocs = location.pathname.startsWith('/docs/reactivesearch');
+		let nestedSidebarFile = {};
+		if (nestedSidebar) nestedSidebarFile = getSidebarFile(nestedSidebar);
 		// Get all nested items and link and make a flat array
 		_.forEach(groups, section => {
```

Comment:
I think we won't need to do this many manipulations we can just change the `groups` based on the docs. Something like this would work: 

```js
const groupsToRender = isRSDocs ? nestedSidebarFile.groups : groups;

_.forEach(groupsToRender, section => {
	_.forEach(section.items, items => {
        ....
	});
});
```